### PR TITLE
Create .gitattributes to auto-normalize eol

### PR DESCRIPTION
**Pull request by hugetim**
I had trouble with my original pull request due to eol problems, so I thought to propose this so others would avoid such troubles.

### CODE REVIEW

**Comment by hugetim** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
I was broken now I'm back. Can I review this? @casey-collab

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# These files are text and should be normalized (convert crlf =&gt; lf)
```
Comment (casey-collab) [inline .gitattributes]: Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.

Comment (casey-collab) [inline .gitattributes]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline .gitattributes]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by hugetim** [comment]
Nevermind.

**Pull request closed without merging**